snooty/rstspec.toml

```diff
@@ -796,15 +796,17 @@ inherit = "_landing-block"
 inherit = "_landing-block"
 help = """Container to hold a group of cards"""
 options.columns = "nonnegative_integer"
+options.style = "string"
```

Comment:
I don't know that this is an appropriate use case for a string -- that prevents us from providing parser validations around its usage. Is there any reason you didn't go with a flag?

Reply:
Yes! Great question. So, I initially went with a flag (`:alt-style:`IIRC), but @schmalliso suggested that we may have more than 2 styles of cards in the future. I felt that naming, rather than flagging, could avoid future tech debt in case we want to expand card styles

Reply:
Ah, gotcha! In that case, this seems like a good use case for a new [enum](https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml#L4-L13).